components/SubgraphEditor/SubgraphConfig/SelectedContract/EventRow.tsx

```diff
@@ -159,7 +170,10 @@ export const EventRow = (props: EventRowProps) => {
         placeholder="Declare event handler function"
       />
       <ActionBtnsContainer>
-        <Search size={16} />
+        <Search
+          size={16}
+          {...(eventHandler.handler && selectedFnExists && { onClick: handleFnLookup })}
```

Comment:
This is hard to read, could this just be written as `onClick={eventHandler.handler && selectedFnExists ? handleFnLookup : undefined}`?

Reply:
I like this format, as it doesn't pass in props unless the condition is true. You can see in my latest commit, that I pass in `className` if it's in a disabled state.

Reply:
Cool, sounds fine!